teamcity.sh

```diff
@@ -2,5 +2,34 @@
 
 set -e
 
-npm install
-npm run build
+setupNvm() {
+  export NVM_DIR="$HOME/.nvm"
+  [[ -s "$NVM_DIR/nvm.sh" ]] && . "$NVM_DIR/nvm.sh"  # This loads nvm
+
+  nvm install
+  nvm use
+}
+
+
+buildJs() {
+  echo "##teamcity[compilationStarted compiler='js']"
+
+  setupNvm
+
+  # clear old packages first
+  rm -rf node_modules
+
+  npm install
+  npm run build
+
+  echo "##teamcity[compilationStarted compiler='js']"
```

Comment:
I assume this should be compilationFinished?

Reply:
Ah yes! Great spot. Fixed now.